### PR TITLE
feat: openapi specs for remove_approvers_from_report_api

### PR DESCRIPTION
**Pull request by sourav014**

### CODE REVIEW

File: src/components/schemas/report.yaml [hunk 1]
```diff
@@ -488,6 +488,25 @@ report_action_with_expenses_ids_to_eject:
           - $ref: './fields.yaml#/id_string'
       example: [ 'txelntgwgwll' ]
 
+remove_approvers_in:
```
Comment (kartikeyrajvaidya) [inline src/components/schemas/report.yaml:491]: remove_approvers_bulk_in

File: src/admin/paths/admin@reports@remove_approvers.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: ../../components/schemas/report.yaml#/remove_approvers_in
+  responses:
+    '200':
```
Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@reports@remove_approvers.yaml:21]: remove approver is not bulk 
so why the response is simple 200
it should ideally be report object

Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@reports@remove_approvers.yaml:21]: remove approver is not bulk
so why the response is simple 200
it should ideally be report object

Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@reports@remove_approvers.yaml:21]: remove approver is not bulk
so why the response is simple 200
it should ideally be report object

Comment (sourav014) [reply src/admin/paths/admin@reports@remove_approvers.yaml:21]: In the docs, it is mentioned that the response of the remove_approvers API will be `200` and `OK`. So I have done it this way in the platform-docs. I will change it to report object

**Review by kartikeyrajvaidya — APPROVED** [review]

**Pull request closed without merging**